test/jdk/sun/security/rsa/SignedObjectChain.java

```diff
@@ -52,13 +53,13 @@ public Test(Chain.SigAlg sigAlg) {
     };
 
     public static void main(String argv[]) {
-        boolean resutl = java.util.Arrays.stream(tests).allMatch(
-                (test) -> Chain.runTest(test));
+        boolean result = Arrays.stream(tests).parallel().allMatch(Chain::runTest);
 
-        if(resutl) {
+        if(result) {
             System.out.println("All tests passed");
         } else {
             throw new RuntimeException("Some tests failed");
         }
     }
 }
+
```

Comment:
Do we need a newline here?

Reply:
+1, would be nice if this extra newline is removed.